core/src/main/scala/org/apache/spark/api/java/JavaSparkContext.scala

```diff
@@ -530,6 +530,7 @@ class JavaSparkContext(val sc: SparkContext)
    * Create an [[org.apache.spark.Accumulator]] integer variable, which tasks can "add" values
    * to using the `add` method. Only the master can access the accumulator's `value`.
    */
+  @deprecated("use AccumulatorV2", "2.0.0")
```

Comment:
Will do. Yes it appears like so in the javadoc --

public Accumulator<java.lang.Integer> intAccumulator(int initialValue, java.lang.String name)
Deprecated. use sc().longAccumulator(String). Since 2.0.0.